test/inductor/test_templated_attention.py

```diff
@@ -49,18 +49,25 @@ def run_test(self, score_mod: Callable, dtype: torch.dtype = torch.float16):
         q = torch.randn((4, 8, 2048, 64), dtype=dtype, device="cuda")
         k = torch.randn((4, 8, 2048, 64), dtype=dtype, device="cuda")
         v = torch.randn((4, 8, 2048, 64), dtype=dtype, device="cuda")
-        ref_out = sdpa_partial(
+        golden_out = sdpa_partial(
             q.to(torch.float64), k.to(torch.float64), v.to(torch.float64)
         )
+        ref_out = sdpa_partial(q, k, v)
         compiled_out = compiled_sdpa(q, k, v)
 
-        tolerance = Tolerances(atol=2e-2, rtol=2e-2)
-        torch.testing.assert_close(
-            ref_out.to(dtype=torch.float32),
-            compiled_out.to(dtype=torch.float32),
-            atol=tolerance.atol,
-            rtol=tolerance.rtol,
-        )
+        compiled_error = (golden_out - compiled_out).abs().mean()
+        ref_error = (golden_out - ref_out).abs().mean()
+        if compiled_error > ref_error * 1.1:
```

Comment:
yup, good change this how we do the rest of the sdpa tests

Reply:
https://github.com/pytorch/pytorch/blob/c9ab9248ce0391c233e7189c0ee71f2aca6a786c/test/test_transformers.py#L93